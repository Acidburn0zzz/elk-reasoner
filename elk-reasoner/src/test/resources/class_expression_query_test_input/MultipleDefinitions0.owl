Prefix(:=<owlapi:ontology#ont11#>)
Prefix(owl:=<http://www.w3.org/2002/07/owl#>)
Prefix(rdf:=<http://www.w3.org/1999/02/22-rdf-syntax-ns#>)
Prefix(xml:=<http://www.w3.org/XML/1998/namespace>)
Prefix(xsd:=<http://www.w3.org/2001/XMLSchema#>)
Prefix(rdfs:=<http://www.w3.org/2000/01/rdf-schema#>)


Ontology(<owlapi:ontology#ont11>

Declaration(Class(<http://example.org/B>))
Declaration(Class(<http://example.org/C>))
Declaration(Class(<http://example.org/D>))
Declaration(ObjectProperty(<http://example.org/R>))

############################
#   Classes
############################

# Class: <http://example.org/B> (<http://example.org/B>)

EquivalentClasses(<http://example.org/B> ObjectSomeValuesFrom(<http://example.org/R> <http://example.org/C>))

# Class: <http://example.org/D> (<http://example.org/D>)

EquivalentClasses(<http://example.org/D> ObjectSomeValuesFrom(<http://example.org/R> <http://example.org/B>))


EquivalentClasses(ObjectSomeValuesFrom(<http://example.org/R> <http://example.org/B>) ObjectSomeValuesFrom(<http://example.org/R> <http://example.org/C>))
)