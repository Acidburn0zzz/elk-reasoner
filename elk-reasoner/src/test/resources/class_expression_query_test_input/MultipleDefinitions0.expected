Prefix(:=<owlapi:ontology#ont12#>)
Prefix(owl:=<http://www.w3.org/2002/07/owl#>)
Prefix(rdf:=<http://www.w3.org/1999/02/22-rdf-syntax-ns#>)
Prefix(xml:=<http://www.w3.org/XML/1998/namespace>)
Prefix(xsd:=<http://www.w3.org/2001/XMLSchema#>)
Prefix(rdfs:=<http://www.w3.org/2000/01/rdf-schema#>)


Ontology(<owlapi:ontology#ont12>

Declaration(Class(<http://example.org/B>))
Declaration(Class(<http://example.org/C>))
Declaration(Class(<http://example.org/D>))
Declaration(ObjectProperty(<http://example.org/R>))

############################
#   Classes
############################

# Class: <http://example.org/B> (<http://example.org/B>)

EquivalentClasses(<http://example.org/B> <http://example.org/D> ObjectSomeValuesFrom(<http://example.org/R> <http://example.org/C>))

# Class: owl:Nothing (owl:Nothing)

SubClassOf(owl:Nothing ObjectSomeValuesFrom(<http://example.org/R> <http://example.org/C>))


SubClassOf(ObjectSomeValuesFrom(<http://example.org/R> <http://example.org/C>) owl:Thing)
)